.changeset/forty-colts-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'browserslist-config-seek': minor
```

Comment:
I'd rather this be a major release. Otherwise consumers of sku could end up with a new browserslist config just from a lockfile update.

Would rather sku's dependency on this be updated in a major version.